pxf/pxf-service/src/main/java/org/apache/hawq/pxf/service/utilities/Utilities.java

```diff
@@ -30,8 +30,27 @@
     public static Object createAnyInstance(Class<?> confClass,
                                            String className, InputData metaData)
             throws Exception {
-        Class<?> cls = Class.forName(className);
+
+        Class<?> cls = null;
+        try {
+            cls = Class.forName(className);
+        } catch (ClassNotFoundException e) {
+            // in case the class name uses the old "com.pivotal.pxf" package
+            // name, recommend using the new package "org.apache.hawq.pxf".
+            if (className.startsWith("com.pivotal.pxf")) {
+                throw new Exception(
+                        "Class "
+                                + className
+                                + " doesn't not appear in classpath. "
+                                + "Try changing the plugin name to start with \"org.apache.hawq.pxf\"",
```

Comment:
If someone uses a custom plugin with their own implementation, it doesn't necessarily need to use this namespace right ? Can we instead mention "Plugins provided by PXF must start with org.apache.hawq.pxf"